packages/devtools_app/lib/src/ui/hover.dart

```diff
@@ -249,16 +327,24 @@ class _HoverCardTooltipState extends State<HoverCardTooltip> {
   /// A timer that removes a [HoverCard] when completed.
   Timer? _removeTimer;
 
-  /// Displays the evaluation result of a source code item.
-  HoverCard? _hoverCard;
+  HoverCard? _currentHoverCard;
```

Comment:
do we need to store this here too since it is also stored on HoverCardTooltipController?